Documentation/cmdref/index.rst

```diff
@@ -13,12 +13,11 @@ Command Reference
       :maxdepth: 0
```

Comment:
IMO it would be better to only list the next level in the Command Reference overview (listing the available CLIs). Showing all their commands doesn't make that much sense :thinking:

Reply:
I agree, but I don't seem to be able to find a way to achieve this 😅. I added `:titlesonly:` which improves the readability of those pages but I don't see how to prevent pages from the sub-indexes from being listed in the toctree of the main index

Reply:
kind of related to this: looks like some command use advanced formatting options in the long help text that appears as titles :confused: 

i wonder whether we should adjust these commands: 

* `cilium`
* `cilium install`
* `cilium  upgrade`

![screenshot-2025-06-13-11-41-00](https://github.com/user-attachments/assets/55932d24-ba7a-41c1-8b62-d3682855b9fa)

Reply:
ah great find, I adjusted the long help texts so they don't get interpreted as titles and render properly as html